Example-Crypto-REXX/RXCRYPT1.rexx

```diff
@@ -0,0 +1,2399 @@
+/*                                                                     */
+/* REFERENCE:                                                          */
+/*     See the z/OS MVS Programming: Callable Services for             */
+/*     High-Level Languages publication for more information           */
+/*     regarding the usage of HWIREST and JSON Parser APIs.            */
+/*                                                                     */
+/*                                                                     */
+/* END OF SPECIFICATIONS  * * * * * * * * * * * * * * * * * * * * * * **/
+
+MACLIB_DATASET = 'SYS1.MACLIB'
+
+TEMP_DATASET = '' /* required input via -D */
+MEMBER_NAME = 'LOCAL' /* default CPC */
+
+TRUE = 1
+FALSE = 0
+
+hwiHostRC = 0     /* HWIHOST rc */
+HWIHOST_ON = FALSE
+PARSER_INIT = FALSE
+ISVREXX = FALSE  /* default to TSO/E, enable via -I */
+VERBOSE = FALSE  /* JSON parser specific, enabled via -V */
+localCPC = TRUE  /* default to lpars on the LOCAL CPC, change via -C */
+
+/* Properties constants */
+cryptoauth = 'crypto-activity-cpu-counter-authorization-control'
+assignedDomainsProp = 'assigned-crypto-domains'
+assignedCryptosProp = 'assigned-cryptos'
+
+/*********************************/
+/* Get program args              */
+/*********************************/
+parse arg argString
+if GetArgs(argString) <> 0 then
+  exit -1
+
+/**************************************/
+/* Before proceeding, ensure the      */
+/* data set specified actually exists */
+/**************************************/
+if VerifyDataSet(TEMP_DATASET) <> 0 then
+  exit fatalErrorAndCleanup('** specified data set does not exist **')
+
+if localCPC then
+  say 'Obtaining LPARs on LOCAL CPC'
+else
+  say 'Starting query of LPARs on CPC:'||CPCname
+
+if ISVREXX then
+  do
+    say 'Running in an ISV REXX environment'
+    hwiHostRC = hwihost("ON")
+    say 'HWIHOST("ON") return code is :('||hwiHostRC||')'
+
+    if hwiHostRC <> 0 then
+      exit fatalErrorAndCleanup('** unable to turn on HWIHOST **')
+    HWIHOST_ON = TRUE
+  end
+
+call IncludeConstants
+
+call JSON_getToolkitConstants
+if RESULT <> 0 then
+  exit fatalErrorAndCleanup( '** Environment error **' )
+
+PROC_GLOBALS = 'VERBOSE parserHandle '||HWT_CONSTANTS
+
+/*********************************/
+/* Create a new parser instance. */
+/*********************************/
+parserHandle = ''
+call JSON_initParser
+if RESULT <> 0 then
+  exit fatalErrorAndCleanup( '** Parser init failure **' )
+
+/*********************************/
+/* Start of BCPii logic          */
+/*********************************/
+if localCPC then
+  do
+    if getLocalCPCInfo() <> 0 then
+      exit fatalErrorAndCleanup( '** failed to get LOCAL CPC info **' )
+  end
+else
+  do
+    if getCPCInfo() <> 0 then
+      do
+        errMsg = '** failed to get CPC info for ('||CPCname||') **'
+        exit fatalErrorAndCleanup(errMsg)
+      end
+    MEMBER_NAME = CPCname
```

Comment:
If you are assigning MEMBER_NAME then the variable name should not be capitalized as this from a programming perspective is synonymous to a constant

Reply:
Fixed.